project/Dependencies.scala

```diff
@@ -137,7 +138,9 @@ object Dependencies {
     COMPILE(
       "org.testcontainers" % "vault" % testcontainersVersion
     ) ++ TEST(
-      "io.rest-assured" % "scala-support" % restAssuredVersion
+      ("io.rest-assured" % "scala-support" % restAssuredVersion)
```

Comment:
Upgrading `restAssured` does not work given new versions are built with Scala 2.13 and this project is still built with Scala 2.12.